ra2ce/analysis/losses/risk_calculation/risk_calculation_cut_from_year.py

```diff
@@ -9,16 +9,12 @@
 )
 
 
-class RiskCalculationCutFromYear(RiskCalculationBase, ABC):
+class RiskCalculationCutFromYear(RiskCalculationBase):
     """
     In this mode, the integration mimics the presence of a flood protection
     """
 
-    def __init__(self, risk_calculation_year: int, losses_gdf: gpd.GeoDataFrame):
-        super().__init__(risk_calculation_year, losses_gdf)
-        self._to_integrate = self._rework_damage_data()
-
-    def _rework_damage_data(self) -> gpd.GeoDataFrame:
+    def _rework_damage_data(self):
```

Comment:
It does not make sense using the `_to_integrate` if you are later going to copy it, you are better just calling the `_get_to_integrate` here (see previous comment).